src/arrayReverse.test.js

```diff
@@ -8,13 +8,23 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(arrayReverse(['qwerty'])).toBeInstanceOf(Array);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([''])).toEqual(['']);
```

Comment:
array with empty string = empty array so it should be [ ] without ' '

Reply:
Nope, empty array is `[]`, and empty array with an empty string is `['']`